tests/src/JIT/Performance/CodeQuality/BenchF/Adams/Adams.cs

```diff
@@ -91,16 +108,36 @@ public static void Test()
             }
         }
     }
+#endif
 
-    private static bool TestBase()
+    [MethodImpl(MethodImplOptions.NoOptimization)]
+    public static int Main(string[] argv)
     {
-        bool result = Bench();
-        return result;
-    }
+        if (argv.Length > 0)
+        {
+            Iterations = Int32.Parse(argv[0]);
+        }
 
-    public static int Main()
-    {
-        bool result = TestBase();
-        return (result ? 100 : -1);
+        Stopwatch sw = Stopwatch.StartNew();
+
+        for (int l = 1; l <= Iterations; l++)
+        {
+            Bench();
+        }
+        sw.Stop();
+
+        Console.WriteLine("Test iterations: {0}; Total time: {1} sec", Iterations, sw.Elapsed.TotalSeconds);
+
+        Console.WriteLine(" BASE.....P1 1/4 (ADAMS-MOULTON), XN = .09999999E +01");
```

Comment:
This output consumes results of the benchmark. otherwise it all can be optimized out. If we'll do so, we'll need to add "volatile" for g_xn, g_yn, g_dn, g_en.